### PR TITLE
Admin panel

### PR DESCRIPTION
**Pull request by a-barzanti**
### Changes

Please describe any changes made in the pull request here.

### Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### Tests
- [ ] Automated tests have been added
- [ ] This PR does not require tests

### CODE REVIEW

File: api/src/accounts/AccountService.ts [hunk 1]
```diff
@@ -44,4 +54,119 @@ export default class AccountService {
...
+        }
+        throw new GenericError(
+            `Account for product ${product} not implemented.`,
+            LogPriority.ERROR,
+        );
+    }
+
+    public async unsubscribeAccount(
+        org: Org,
+        account: TagManagerAccount | DataManagerAccount | undefined,
```
Comment (cwbeck) [inline api/src/accounts/AccountService.ts:77]: why are we testing for undefined?

Comment (a-barzanti) [reply api/src/accounts/AccountService.ts:77]: moving the check to this method will simplify the call:
`                return this.accountService.unsubscribeAccount(
                    org,
                    await this.accountService.getAccountByProduct(org, data.product),
                );`
                
if we want to check at the moment we call we need to use a support constant and throw before the call.
It just look cleaner this way IMO.

Comment (a-barzanti) [reply api/src/accounts/AccountService.ts:77]: Alternatively we need a _getAccountByProductThrows_

Comment (a-barzanti) [reply api/src/accounts/AccountService.ts]: Fixed

**Pull request merged**